src/app/(admin)/settings/components/sections/organization.tsx

```
@@ -0,0 +1,224 @@
+    manager: string;
+}
+
+interface Region {
+    id: string;
+    name: string;
+    country: string;
+    timezone: string;
+}
+
+export function OrganizationSettings() {
+    const [departments, setDepartments] = useState<Department[]>([
```

ESLint

[error] 'setDepartments' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)